docs/designs/012-workflow-XID-13-and-31.md

```
@@ -0,0 +1,417 @@
+## XID 13
+XID 13 (Graphics Engine Exception) logged for general user application faults. Typically this is an out-of-bounds error where the user has walked past the end of an array, but could also be an illegal instruction, illegal register, or other case.errors.
+
+It can indicate different root causes depending on:
+- Whether they repeat on the same hardware components (GPC/TPC/SM)
+- Whether they occur in isolation or as part of a burst
+
+A single recommended action is insufficient - we need to have context-aware decision making.
+
+Workflow XID 13 is defined like this: 
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)